mediapipe/tasks/python/components/proto/segmenter_options.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Segmenter options data class."""
+
+import enum
+
+
```

Comment:
Thanks for your quick update! Please move `OutputType` and `Activation` to image_segmenter.py too.

Reply:
@lu-wang-g I've moved them now.